admin/jobs/river/org_jobs.go

```diff
@@ -173,3 +173,77 @@ func (w *DeleteOrgWorker) Work(ctx context.Context, job *river.Job[DeleteOrgArgs
 
 	return nil
 }
+
+type LogInactiveOrgsArgs struct{}
+
+func (LogInactiveOrgsArgs) Kind() string { return "log_inactive_orgs" }
+
+type LogInactiveOrgsWorker struct {
+	river.WorkerDefaults[LogInactiveOrgsArgs]
+	admin  *admin.Service
+	logger *zap.Logger
+}
+
+func (w *LogInactiveOrgsWorker) Work(ctx context.Context, job *river.Job[LogInactiveOrgsArgs]) error {
```

Comment:
Since this job actually hibernates projects, I think a better name would be `HibernateInactiveOrgs`